tests/examples/awful/wibar/left.lua

```diff
@@ -0,0 +1,10 @@
+--DOC_NO_USAGE --DOC_GEN_IMAGE
+local awful     = require("awful") --DOC_HIDE
+local wibox     = require("wibox") --DOC_HIDE
+
+screen[1]._resize {width = 480, height = 200} --DOC_HIDE
+
+    local wb = awful.wibar { position = "left" }
```

Comment:
I think it would be better if you also add widgets setup for the generated image (maybe `--Doc_HIDE` the `wb:setup()`, tho).

Reply:
Added the widgets as requested